### ISSUE TITLE
Fluent UI Designer, colors shown are not reflective of the color input values, accessibility checker fails to report color contrast issues.

### ISSUE THREAD

**Issue by Priyanka-Chauhan123** [microsoft/fluentui#16543]
Fluent UI Designer's accessibility checker fails to report color contrast issues.  But it is hard to tell whether it is an accChecker issue or the Designer tool's issue because the colors kept changing even with the same color inputs.

**Repro steps:**
1. Open Fluent UI Theme Designer (windows.net)
2. Enter these values in the primary color, Text color and Background color text boxes.
Primary color #ffc83d, Text color #f3f2f1, Background color #1b1a19
3. Note that a numbers of the UI items seem to be below minimum color contrast ratio, but the accessibility checker was showing no issues.
4. If you were to delete the last digit of the color code, then retype it, the colors in the designer change.
5. Even with the default color scheme that comes with the tool (the initial text color was set to #323130 when the page first opens), by deleting the last digit "0", then retype "0", the accessibility checker result changed from pass to fail.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Ba73546c7-765f-489c-b18f-afb659fe99e6%7D&cid=76632f93-4480-42f8-9090-a75b9ca71916

**Repro video attached.**
https://user-images.githubusercontent.com/66783558/105136378-8cd3d380-5b17-11eb-999a-932b8a62eb5b.mp4

**Comment by layershifter** [comment]
@Priyanka-Chauhan123 I was able to reproduce both issues (controls below minimum color contrast ratio and issue from #4).

@JustSlone can you please suggest who can help with this issue?
